### PR TITLE
add a protobuf fork for int64/uint64 as int marshaling support

### PR DESCRIPTION
**Pull request by luciferinlove**
| Q               | A
| --------------- | ---
| Bug fix?        | no|
| New feature?    | no|
| API breaks?     | no|
| Deprecations?   | no|
| Related tickets |
| License         | Apache 2.0

int64/uint64 as int marshaling support for the updated protobuf

### CODE REVIEW

File: scripts/download-deps.sh [hunk 1]
```diff
@@ -69,9 +69,9 @@ ensure-binary-version controller-gen ${controller_gen_version} "sigs.k8s.io/cont
 ensure-binary-version cue-gen ${istio_deps_replacement_version} "istio.io/tools" "/cmd/cue-gen" "${istio_tools_replacement_module_name}"
 ensure-binary-version protoc-gen-go-grpc ${golang_protoc_grpc_version} "google.golang.org/grpc" "/cmd/protoc-gen-go-grpc"
 ensure-binary-version protoc-gen-go ${golang_protoc_gen_go_version} "google.golang.org/protobuf" "/cmd/protoc-gen-go"
-ensure-binary-version protoc-gen-golang-deepcopy ${istio_deps_version} "istio.io/tools" "/cmd/protoc-gen-golang-deepcopy"
-ensure-binary-version protoc-gen-golang-jsonshim ${istio_deps_version} "istio.io/tools" "/cmd/protoc-gen-golang-jsonshim"
-ensure-binary-version protoc-gen-docs ${istio_deps_version} "istio.io/tools" "/cmd/protoc-gen-docs"
+ensure-binary-version protoc-gen-golang-deepcopy ${istio_deps_version} "github.com/luciferinlove/istio-tools" "/cmd/protoc-gen-golang-deepcopy"
+ensure-binary-version protoc-gen-golang-jsonshim ${istio_deps_version} "github.com/luciferinlove/istio-tools" "/cmd/protoc-gen-golang-jsonshim"
+ensure-binary-version protoc-gen-docs ${istio_deps_version} "github.com/luciferinlove/istio-tools" "/cmd/protoc-gen-docs"
```
Comment (pregnor) [inline scripts/download-deps.sh:74]: **Question**: Why are the custom deps downloads section is required? I'm not familiar what brings the custom istio-tools into the picture, I thought only the golang/protobuf replace needs to be done, the marshaler config and the regeneration of the files.

**Review by pregnor — COMMENTED** [review]
LGTM, but I'm no expert here

File: scripts/download-deps.sh (hunk 1 above)
Comment (luciferinlove) [reply scripts/download-deps.sh:74]: There is a [protoc-gen-golang-jsonshim](https://github.com/LuciferInLove/istio-tools/tree/master/cmd/protoc-gen-golang-jsonshim) update that adds the `Int64Uint64asIntegers: true` option to marshalers during build.

**Review by nishantapatil3 — APPROVED** [review]
lgtm!

**Pull request merged**